blog/_posts/2018-08-08-one-point-zero.md

```diff
@@ -0,0 +1,161 @@
+ title: "Julia 1.0"
+ snippet: "Announcing the release of Julia 1.0"
+---
+
+The much anticipated 1.0 release of [Julia](https://julialang.org) is the culmination of
+nearly a decade of work to build a language for greedy programmers. Julia was [first
+publicly announced](https://julialang.org/blog/2012/02/why-we-created-julia) with a number
+of strong demands on the language:
+
+> We want a language that’s open source, with a liberal license. We want the speed of C with
+> the dynamism of Ruby. We want a language that’s homoiconic, with true macros like Lisp,
+> but with obvious, familiar mathematical notation like Matlab. We want something as usable
+> for general programming as Python, as easy for statistics as R, as natural for string
+> processing as Perl, as powerful for linear algebra as Matlab, as good at gluing programs
+> together as the shell. Something that is dirt simple to learn, yet keeps the most serious
+> hackers happy. We want it interactive and we want it compiled.
+
+A vibrant and thriving community has grown up around this language, with people from all
+around the world iteratively refining and shaping Julia in pursuit of that goal. Over 700
+people have contributed to Julia itself and even more people have made thousands of amazing
+open source Julia packages. All told, we have built a language that is:
+
+* **Fast**: Julia was designed from the beginning for high performance. Julia programs
+  compile to efficient native code for multiple platforms via LLVM.
+* **General**: It uses multiple dispatch as a paradigm, making it easy to express many
+  object-oriented and functional programming patterns. The standard library provides
+  asynchronous I/O, process control, logging, profiling, a package manager, and more.
+* **Dynamic**: Julia is dynamically-typed, feels like a scripting language, and has good
+  support for interactive use.
+* **Technical**: It excels at numerical computing with a syntax that is great for math, many
+  supported numeric data types, and parallelism out of the box. Julia's multiple dispatch
+  is a natural fit for defining number and array-like data types.
+* **Optionally typed**: Julia has a rich language of descriptive data types, and type
+  declarations can be used to clarify and solidify programs.
+* **Composable**: Julia’s packages naturally work well together. Matrices of unit
+  quantities, or data table columns of currencies and colors, just work — and with good
+  performance.
+
+Try Julia by [downloading version 1.0 now](https://julialang.org/downloads/). If you’re
+upgrading code from Julia 0.6 or earlier, we encourage you to first use the transitional 0.7
+release, which includes deprecation warnings to help guide you through the upgrade process.
+Once your code is warning-free, you can change to 1.0 without any functional changes. The
+package ecosystem is in the midst of taking advantage of this stepping stone and many
+packages are already 1.0-compatible! This includes ...
```

Comment:
TODO: add list of packages that are already 1.0 compatible.

Reply:
I assume this will be a URL, so that we can keep updating it without changing the post.

Reply:
I had been thinking about an immediate list, but a dynamic page is even better.  The best answer is an output from NewPkgEval, but I don't want to put _anything_ more on Keno's plate.  I'll look into what we can do here.

Reply:
I can give you a JSON file with the NewPkgEval results. You can do whatever with that.

Reply:
That'd be great.  I think we can create a simple page that simply lists the passing packages, sorted by their number of stars.